### ISSUE TITLE
Doclet is suppressing the Javadoc Warnings and Errors

### ISSUE THREAD

**Issue by DJViking** [jtulach/codesnippet4javadoc#16]
This doclet is suppressing the standard Javadoc warnings and errors.

Discovered this when I noticed Java 11 complained about some HTML tags that worked fine when we where using Java 8. We did not get them in the API where we are using this doclet, but got a lot of them in our application. I found this strange, so I disabled this doclet and I got the same warnings and errors in our API.

With this doclet configured I am not getting the javadoc warnings and errors. 
Such like these:
```
error: tag not supported in the generated HTML version: tt
warning: no @return
warning: no @param for <C>
```

```
javadoc {
    options.doclet = "org.apidesign.javadoc.codesnippet.Doclet"
    options.docletpath = configurations.snippetdoclet.files.asType(List)
    options.addStringOption "snippetpath", "src/test/java"
    options.addStringOption "maxLineLength", "120"
    options.addStringOption "Xmaxerrs", "400"
    options.addStringOption "Xmaxwarns", "400"
    options.addStringOption "--add-opens=jdk.javadoc/com.sun.tools.javadoc.main=ALL-UNNAMED"
}
```

Also with this doclet I cannot write short links in the JavaDoc.
warning - Tag @link: reference not found
Using this doclet it complains about @{link Class}, even though Class is on the imports, but the warning goes away when I use @{link com.company.Class}
Not using this doclet Javadoc does not complain about this short link.

**Comment by jtulach** [comment]
Interesting. Sample project showing the problem would be helpful.

**Comment by DJViking** [comment]
I have a sample project. I will publish it to GitHub soon.

**Comment by DJViking** [comment]
A small test project
https://github.com/DJViking/test-api

Remove the javadoc doclet configuration from build.gradle to see the Javadoc errors and warnings that are otherwise suppressed when using the doclet.

```
> Task :javadoc FAILED
/home/sverre/workspace-github/test-api/src/main/java/test/api/Library.java:17: error: tag not supported in the generated HTML version: tt
     * @return <tt>true</tt>
               ^
/home/sverre/workspace-github/test-api/src/main/java/test/api/Library.java:32: warning: no description for @param
     * @param value
```

**Comment by JaroslavTulach** [comment]
I acknowledge the problem.

**Comment by jtulach** [comment]
Fixed in version 0.50. Thanks for the report!
